connect/src/com/telenor/connect/AbstractSdkProfile.java

```diff
@@ -166,6 +166,10 @@ protected void previewParameters(ParametersHolder parameters) {
                         && isEmpty(wellKnownConfig.getNetworkAuthenticationTargetUrls()))) {
                     parameters.add("login_hint", String.format("MCCMNC:%s", mccMnc));
         }
+
+        if (!ConnectSdk.isCellularDataNetworkConnected()) {
```

Comment:
Are you sure about isCellularDataNetworkConnected? Works fine on my device, which indicates the issue you experience might not be by design anyway. But, since it's obviously you to have a device where this method does not work as it should, could you do me a favour? Could you test the same method on your device, but this time using PR 67? It contains another version of this method, which I tested and pushed it for review yesterday. If your test succeeds, I may replay the changes made in 67 in this PR.